database/sql/V029__create_onchain_placed_orders.sql

```diff
@@ -0,0 +1,20 @@
+-- Table to store all orders placed via any on-chain broadcaster contract
+-- uid is needed as unique identifier for the order
+-- Sender is the address who called the broadcasting contract. For ethflow orders, this would be the user placing the order
+-- is_reorged is a flag indicated whether the order was placed and then reorged.
+-- block_number + log_index are used to deal with chain reverts.
+-- All other information about the order is directly translated into an order and the information is stored in the orders table
+
+CREATE TABLE onchain_placed_orders (
+    uid bytea PRIMARY KEY,
```

Comment:
I believe `PRIMARY KEY` means that the key is also unique.
What happens if an order is reorged and later the order is recreated? In principle the other parameters could be different after a reorg but have the same `uid`. Is the reorged order going to be overwritten?

Reply:
To answer myself, the order is overwritten, which makes sense to me:
https://github.com/cowprotocol/services/pull/500/files#diff-38c963ec3bb54b2ecd38df09647b4f786827c6d72e86aa68fe6694bfd35f5525R47-R64